io/zenoh-transport/tests/unicast_concurrent.rs

```diff
@@ -412,3 +412,37 @@ fn transport_ws_concurrent() {
         transport_concurrent(endpoint01, endpoint02).await;
     });
 }
+
+#[cfg(feature = "transport_shm")]
+#[test]
```

Comment:
I believe there is no need to run this test every time on the CI (it will make increase its duration).
Mark the test as follows:
```
#[test]
#[ignore]
```
It can be then run with `cargo test -- --ignored`.

Please apply it as well to `transport_ws_concurrent`, which slipped in from the past.